optimum/exporters/neuron/convert.py

```diff
@@ -434,6 +438,8 @@ def export_neuronx(
         compiler_args.extend(["--auto-cast-type", auto_cast_type])
     else:
         compiler_args = ["--auto-cast", "none"]
+    # WARNING: Enabled experimental parallel compilation
+    compiler_args.extend(["--enable-experimental-O1"])
```

Comment:
should we add `f"--num-parallel-jobs {os.cpu_count()}"` to maximize the compilation? 

https://awsdocs-neuron.readthedocs-hosted.com/en/latest/release-notes/compiler/neuronx-cc/index.html?highlight=enable-experimental-O1#id1

Reply:
I did not see any improvement on the compilation time with the flag neither.